pkgs/by-name/uv/uv/package.nix

```diff
@@ -22,7 +20,7 @@ python3Packages.buildPythonApplication rec {
     hash = "sha256-Kieh76RUrDEwdL04mvCXOqbXHasMMpXRqp0LwMOIHdM=";
   };
 
-  cargoDeps = rustPlatform.importCargoLock {
+  cargoLock = {
```

Comment:
Now we can use the brand new fetcher that allows us to get rid of our vendored `Cargo.lock`:
```nix
useFetchCargoVendor = true;
cargoHash = "";
```

And of course, `git rm pkgs/by-name/uv/uv/Cargo.lock`

Reply:
Not to keen on doing an unrelated change & build testing again.

Reply:
As you want. I can do it in a second time.